config.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+class Config:
```

Comment:
ok, so this class currently holds some configs and some not (receiver mail). I also don't understand why we limit ourselves to gmail (why not simply config mail and smtp?) Also this class has currently zero responsibility (it's a dictionary). If you plan to further expand it, mb leave a TODO comment or something.

Reply:
1. Yes, i should add more values here
2. Using gmail account was actually the simplest way to do that. of course it reasonable to change, but can be done in future changes.
3. I think making config as class instead of dictionary just more readable, clear and easier to use. I am not planning to add extra functionality here.

Reply:
this class mb could read config from file and store it

Reply:
I'll think about it.